CHANGELOG.md

```diff
@@ -11,6 +14,11 @@
 * replace babel-eslint with @babel/eslint-parser. Refs UIOAIPMH-52.
 * Remove react-hot-loader from package.json. Refs UIOAIPMH-51.
 
+## [3.2.0] (https://github.com/folio-org/ui-oai-pmh/tree/v3.2.0) (2022-07-07)
```

Comment:
9 and 17 lines are the same, looks strange for changelog. Take a look plz

Reply:
@AndreiShumski2 Ye thanks a lot! It was my mistake.